docs/apache-airflow/best-practices.rst

```diff
@@ -109,6 +109,9 @@ or if you need to deserialize a json object from the variable :
 
     {{ var.json.<variable_name> }}
 
+Alternative option is to read from the environment variables in the top level python code instead of airflow variables.
+This will avoid new connections to airflow metadata DB everytime Airflow parses the python file.
+
```

Comment:
I think it's better to place this text in the following paragraph "Top level Python Code" because you mention in the explanation "top level python code" but it's not really defined till the user read the following paragraph. WDYT?

Reply:
I think it's pretty understandable :D and it is right after, so I think I am happy with the way how it is now.